pkg/sensors/tracing/kprobe_net_test.go

```diff
@@ -1326,3 +1376,148 @@ spec:
 	err := jsonchecker.JsonTestCheckExpectWithKeep(suite.T(), checker, false, false)
 	suite.Require().NoError(err)
 }
+
+func (suite *KprobeNet) TestKprobeSocketAndSockaddrUn() {
+	hookFull := `apiVersion: cilium.io/v1alpha1
+kind: TracingPolicy
+metadata:
+  name: "security-socket-connect-unix"
+spec:
+  kprobes:
+  - call: "security_socket_connect"
+    syscall: false
+    args:
+    - index: 0
+      type: "socket"
+    - index: 1
+      type: "sockaddr_un"
+    selectors:
+    - matchArgs:
+      - index: 0
+        operator: "Family"
+        values: ["AF_UNIX"]
+      - index: 1
+        operator: "Family"
+        values: ["AF_UNIX"]
+      - index: 1
+        operator: "Prefix"
+        values: ["@"]
+    - matchArgs:
+      - index: 0
+        operator: "Family"
+        values: ["AF_UNIX"]
+      - index: 1
+        operator: "Family"
+        values: ["AF_UNIX"]
+      - index: 1
+        operator: "Prefix"
+        values: ["/"]
```

Comment:
The selectors matching index 1 will match everything because there is no specific matching in bpf/process/types/basic.h.